Sources/CassandraClient/Data+PaginatedRows.swift

```diff
@@ -0,0 +1,245 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Cassandra Client open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift Cassandra Client project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift Cassandra Client project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@_implementationOnly import CDataStaxDriver
+import Logging
+import NIO
+
+extension CassandraClient {
+    /// Resulting row(s) of a Cassandra query. Data are paginated.
+    public final class PaginatedRows {
+        let session: Session
+        let statement: Statement
+        let eventLoop: EventLoop?
```

Comment:
nit: `eventLoop` always seems to be available through the initializer?

```suggestion
        let eventLoop: EventLoop
```

Reply:
Oh, I see a different initializer down there. Nevermind.